test/arithmetic/fixed_sqrt_test.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Indentifier: BSD-3-Clause
+//
+// fixed_sqrt_test.dart
+// Tests for fixed-point square root.
+//
+// 2025 March 5
+// Authors: James Farwell <james.c.farwell@intel.com>,
+// Stephen Weeks <stephen.weeks@intel.com>,
+// Curtis Anderson <curtis.anders@intel.com>
+
+import 'dart:math';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/src/arithmetic/arithmetic.dart';
+import 'package:test/test.dart';
+
+void main() {
+  tearDown(() async {
+    await Simulator.reset();
+  });
+  test('sqrt(1)', () async {
```

Comment:
Its useful to leave a singleton around to help test one number, but the rest of the tests could be in a loop:
```
final tests = [1.5, 1.7, ...];
for (final t in tests) {
    fixed.put(FixedPointValue.ofDouble(t,